resources/schemas/dbscripts/postgresql/targetedms-21.002-21.003.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE targetedms.QCMetricConfiguration ADD COLUMN TraceValue REAL;
+ALTER TABLE targetedms.QCMetricConfiguration ADD COLUMN TimeValue REAL;
+ALTER TABLE targetedms.QCMetricConfiguration ADD COLUMN Trace BIGINT;
```

Comment:
As we discussed, this should be the trace's name, not a FK the SampleFileChromInfo.